test/fileSummaryV1/integration_fileSummaryV1.test.js

```diff
@@ -0,0 +1,162 @@
+            if (error) { t.log(response.body) };
+            t.end(error);
+        });
+});
+
+test.cb('integration_fileSummaryV1 responds with 200 and file metadata but no signed url if dos uri is valid, but an invalid bearer token is provided', (t) => {
+    supertest
+        .post('/fileSummaryV1')
+        .set('Content-Type', 'application/json')
+        .set('Authorization', `Bearer badToken`)
+        .send({ uri: 'dos://dg.4503/preview_dos.json'})
+        .expect((response) => {
+            assert.strictEqual(response.statusCode, 200, "Incorrect status code");
+            assert(response.body.bucket, "fileSummary did not return metadata");
+            assert(!response.body.signedUrl, "fileSummary returned a signed url but it should not have");
+        })
+        .end((error, response) => {
+            if (error) { t.log(response.body) };
+            t.end(error);
+        });
+});
+
+test.cb('integration_fileSummaryV1 responds with 200 and file metadata but no signed url if dos uri is valid, a valid bearer token is provided, but account is not linked with provider', (t) => {
+    supertest
+        .post('/fileSummaryV1')
+        .set('Content-Type', 'application/json')
+        .set('Authorization', `Bearer ${unauthorizedToken}`)
+        .send({ uri: 'dos://wb-mock-drs-dev.storage.googleapis.com/preview_dos.json'})
+        .expect((response) => {
+            assert.strictEqual(response.statusCode, 200, "Incorrect status code");
+            assert(response.body.bucket, "fileSummary did not return metadata");
+            assert(!response.body.signedUrl, "fileSummary returned a signed url but it should not have");
+        })
+        .end((error, response) => {
+            if (error) { t.log(response.body) };
+            t.end(error);
+        });
+});
+
+test.cb('integration_fileSummaryV1 responds with 200, file metadata, and a signed url if dos uri is valid and valid authorization is provided', (t) => {
+    supertest
+        .post('/fileSummaryV1')
+        .set('Content-Type', 'application/json')
+        .set('Authorization', `Bearer ${authorizedToken}`)
+        .send({ uri: 'dos://dg.4503/preview_dos.json'})
+        .expect((response) => {
+            assert.strictEqual(response.statusCode, 200, "Incorrect status code");
+            assert(response.body.bucket, "fileSummary did not return metadata");
+            assert(response.body.signedUrl, "fileSummary did not return a signed url");
+        })
+        .end((error, response) => {
+            if (error) { t.log(response.body) };
+            t.end(error);
+        });
+});
+
+test.cb('integration_fileSummaryV1 responds with 502 and unauthorized response if gs uri is valid, but an invalid bearer token is provided', (t) => {
+    supertest
+        .post('/fileSummaryV1')
+        .set('Content-Type', 'application/json')
+        .set('Authorization', `Bearer badToken`)
+        .send({ uri: 'gs://wb-mock-drs-dev/public/dos_test.txt'})
```

Comment:
I see you're using different uris in the different tests.  I think when testing different flavors of the same test, it's worth using the same uri, just to be scientific and only change 1 variable at a time.  

That being said, this `gs://` uri reminds me that the Martha logic does different things when it detects a `dos://` uri or a `gs://` uri.  In other words, there are probably several tests in here that we might want to test with each type of uri.  I think these would be limited to the successful `200` responses, but there might be a few of the failure tests that we want to check to.  You might be able to get a sense of what to test if you take a look at the code and look where it diverges based on `gs://` vs. `dos://`.  I'm happy to sit with you too and we can see if we can identify which tests should test both protocols.